### ISSUE TITLE
Merge the Schema Simplification Project

### ISSUE THREAD

**Issue by jmoldow** [learning-unlimited/ESP-Website#509]
This involves doing four things:
- fixing navbar categories to not use the data tree
- fixing teacher training / interview signups to not use the data tree and user bits
- fixing the accounting modules to not use the data tree (this will probably require rewriting all of the accounting modules)
- testing

In theory, we could merge the project in its current state (after testing) and then get to the other tickets later.

We shouldn't merge into main until we can guarantee enough server downtime to run the database migrations.
